### PR TITLE
Cppcourse

### PR DESCRIPTION
**Pull request by Ajay-singh1**
## What does this PR do?
Add resource(s) 

## For resources
Youtube
### Description
Added C++ Course in the newly created Hindi/Urdu File
### Why is this valuable (or not)?
It is very valuable for those interested in Computer Science Engineering or particularly Computer Science.
### How do we know it's really free?
Beacuse it is available on youtube
### For book lists, is it a book? For course lists, is it a course? etc.
It is a course
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Ajay-singh1** [comment]
@eshellman I will try to work on the Hindi/Urdu File.

File: courses/free-courses-ur.md [hunk 1]
```diff
@@ -12,3 +14,14 @@
...
 * [HTML5 & CSS3 Tutorials In Urdu and Hindi](https://youtube.com/playlist?list=PLUyYwyJA_WfTr3YWWJ41_V7TrRZoq6cBT) - ProgramInUrdu
+
+
+### Javascript
+
+* [Javascript Tutorials In Urdu and Hindi](https://www.youtube.com/watch?v=Lgxgm-T9cgA&list=PL0b6OzIxLPbx-BZTaWu_AF7hsKo_Fvsnf) - Yahoo Baba
+* [Learn JavaScript in Urdu & Hindi - For Beginners - Introduction](https://www.youtube.com/watch?v=NKgdmmiPVUc&list=PL52B8F99BC6AD1647) - WP Academy
+
+
+### C++
```
Comment (eshellman) [inline courses/free-courses-ur.md:25]: need a link anchor here. see other lists

File: courses/free-courses-ur.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 * [C#](#csharp)
 * [HTML and CSS](#html-and-css)
+* [Javascript](#javascript)
+* [C++](#cplusplus)
```
Comment (eshellman) [inline courses/free-courses-ur.md:6]: should be in alphabetical order

File: courses/free-courses-ur.md [hunk 3]
```diff
@@ -12,3 +14,14 @@
 ### HTML and CSS
 
 * [HTML5 & CSS3 Tutorials In Urdu and Hindi](https://youtube.com/playlist?list=PLUyYwyJA_WfTr3YWWJ41_V7TrRZoq6cBT) - ProgramInUrdu
+
+
+### Javascript
+
+* [Javascript Tutorials In Urdu and Hindi](https://www.youtube.com/watch?v=Lgxgm-T9cgA&list=PL0b6OzIxLPbx-BZTaWu_AF7hsKo_Fvsnf) - Yahoo Baba
```
Comment (eshellman) [inline courses/free-courses-ur.md:21]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

File: courses/free-courses-ur.md [hunk 4]
```diff
@@ -12,3 +14,14 @@
...
+* [Learn JavaScript in Urdu & Hindi - For Beginners - Introduction](https://www.youtube.com/watch?v=NKgdmmiPVUc&list=PL52B8F99BC6AD1647) - WP Academy
```
Comment (eshellman) [inline courses/free-courses-ur.md:22]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

**Review by eshellman — CHANGES_REQUESTED** [review]
if you add something here that's also in the hindu list, you should remove it from there

**Comment by Ajay-singh1** [comment]
@eshellman Kindly Review the changes and suggest me why the lint is fauiling.

**Comment by Ajay-singh1** [comment]
@eshellman We are good to go!

File: courses/free-courses-ur.md (hunk 2 above)
Comment (eshellman) [reply courses/free-courses-ur.md:4]: `C` comes before `H`

**Comment by Ajay-singh1** [comment]
@eshellman I think I have opened duplicate PRs this PR will solve #10407 and #10418 , I am closing both the PRs and work on this one only.Will remind you once after the work is done.

**Comment by eshellman** [comment]
Please delete the duplicate course in the hindi list

**Comment by Ajay-singh1** [comment]
@eshellman Which Course needs to be deleted kindly tell me as I have checked the Hindi List and Didn't Find Any Matching Course.

**Comment by Ajay-singh1** [comment]
@eshellman I have deleted the duplicate course of javascritp in Hindi List are we ready to merge now.

**Comment by Ajay-singh1** [comment]
@eshellman Kindly Review the PR and merge it!

**Comment by Ajay-singh1** [comment]
@Thenlie Can you review this PR and suggest any changes to be made?

File: courses/free-courses-ur.md [hunk 5]
```diff
@@ -1,14 +1,25 @@
 ### Index
 
+* [C++](#cplusplus)
 * [C#](#csharp)
 * [HTML and CSS](#html-and-css)
 
 
+### <a id="cplusplus"></a>C++
+
+* [Programming Fundamentals With C++ Complete Course In urdu | Hindi](https://www.youtube.com/playlistlist=PL4QkPoTgwFULciDFVJEHEwOKMtf9Q_Aqh) - Kacs Learnings
```
Comment (eshellman) [inline courses/free-courses-ur.md:10]:
```suggestion
* [Programming Fundamentals With C++ Complete Course In urdu | Hindi](https://www.youtube.com/playlist=PL4QkPoTgwFULciDFVJEHEwOKMtf9Q_Aqh) - Kacs Learnings
```

**Comment by Ajay-singh1** [comment]
@eshellman Kindly Review this! I wish to work on another issue.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**